sonar-db/src/main/java/org/sonar/db/purge/PurgeMapper.java

```diff
@@ -53,14 +50,10 @@
 
   void updatePurgeStatusToOne(@Param("analysisUuids") List<String> analysisUuid);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)